test/cql-pytest/test_bad_grammar.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+#############################################################################
+# Tests for the parser, focusing on rejected grammar
+#############################################################################
+
+import pytest
+from util import new_test_table
+from cassandra.protocol import InvalidRequest, SyntaxException
+
+
+# A collection of tests for grammar that should be rejected, but wasn't at
+# some point in the past.
```

Comment:
‎These two lines are a general summary of this test file, which repeats the general summary you have above between the "######" lines. Please pick one version and remove the other one.

Reply:
done